twitter_acquier.rb

```diff
@@ -0,0 +1,47 @@
+class TwitterAcquier
+  def session_to_google_auth
+    credentials = Google::Auth::UserRefreshCredentials.new(
+      client_id: ENV['GOOGLE_CLIENT_ID'],
+      client_secret: ENV['GOOGLE_CLIENT_SECRET'],
+      scope: [
+        "https://www.googleapis.com/auth/drive",
+        "https://spreadsheets.google.com/feeds/",
+      ],
+      refresh_token: ENV['GOOGLE_REFRESH_TOKEN'])
+    session = GoogleDrive::Session.from_credentials(credentials)
+    session
+  end
+
+  def find_work_sheet(session, sp_url)
+    sp = session.spreadsheet_by_url(sp_url)
+    ws = sp.worksheet_by_title("test_volley")
+    ws
+  end
+
+  def send_data_to_sp(analytics_file, ws)
```

Comment:
[IMO] `sp` is not each to read. I think `spreadsheet` or `sheet`.. are better. `ws` is also the same.